x/evm/core/vm/custom_eip_testing.go

```diff
@@ -6,14 +6,6 @@
 
 package vm
 
-import (
```

Comment:
nit: I would add a bit of context why we have this file. Something like:

// This file is used to allow the testing of EVM configuration initialization
// without the need to introduce testing requirements in the final binary. In
// this case, the file provides the possibility to restore the EIP activator
// functions to the initial state without the need to compile `ResetActivators`
// in the final binary.